### DISCUSSION TITLE
Device screen is not showing. @yume-chan version 0.0.19 .

### DISCUSSION THREAD

**Discussion by tiennguyen12g** [yume-chan/ya-webadb#545] [General]

**Comment by tiennguyen12g** [comment]


**Reply by yume-chan** [reply]
Your React code is fundamentally wrong, I have nothing to comment.

**Comment by yume-chan** [comment]
0.0.19 doesn't support Scrcpy 2.0, it also doesn't have exports like `AdbScrcpyOptionsLatest` and `ScrcpyOptionsLatest`, I have no idea where you got those.

**Comment by tiennguyen12g** [comment]
You said "AdbScrcpyOptionsLatest and ScrcpyOptionsLatest" does not have export, but i see in your code you use it?
https://github.com/yume-chan/ya-webadb/blob/db25dd9161434ad1f29c5514a465d65286af282f/apps/demo/src/components/scrcpy/state.tsx#L3
https://github.com/yume-chan/ya-webadb/blob/db25dd9161434ad1f29c5514a465d65286af282f/apps/demo/src/components/scrcpy/state.tsx#L20
https://github.com/yume-chan/ya-webadb/blob/db25dd9161434ad1f29c5514a465d65286af282f/apps/demo/src/components/scrcpy/state.tsx#L258

"0.0.19 doesn't support Scrcpy 2.0". I find in node_modules/@yume-chan/scrcpy/bin/version.js, the version is 2.0. And in the fetch-server, you use this version.
https://github.com/yume-chan/ya-webadb/blob/db25dd9161434ad1f29c5514a465d65286af282f/apps/demo/src/components/scrcpy/fetch-server.ts#L3

**Reply by yume-chan** [reply]
`main` branch is the development branch. Use v0.0.19 tag for released version. https://github.com/yume-chan/ya-webadb/tree/v0.0.19

**Reply by tiennguyen12g** [reply]
Thank bro.
